src/content/en/fundamentals/instant-and-offline/web-storage/index.md

```diff
@@ -223,6 +237,6 @@ of interest:
 
 * Deep Dive: IndexedDB (coming soon)
 
-* Deep Dive: Cache API (coming soon)
+* [Deep Dive: Cache API](cache-api)
```

Comment:
So I only just now saw that we have a list of docs that are coming soon-- this is weird to me. Just how long does coming soon imply? I'd be inclined to delete the whole list, and just link to Cache API doc, as mentioned in other comment, above.